src/components/Button/Button.styles.js

```diff
@@ -0,0 +1,33 @@
+import { css } from '@emotion/core'
+import { buttonReset, Color, Radius } from 'lib/theme'
+import { type, radius } from './constants'
+
+const styles = {
+  root: css({
+    ...buttonReset,
+    height: 65,
+    width: '100%',
+    ':disabled': {
+      opacity: 0.25
+    }
+  }),
+  [radius.ROUNDED]: css({
+    borderRadius: Radius.ROUNDED
+  }),
+  [radius.PILL]: css({
+    borderRadius: Radius.PILL
+  }),
+  [type.FILL]: css({
+    color: Color.WHITE,
+    background: '#FF5A5F',
```

Comment:
will you update this to use the coral const?

Reply:
can i sneak it into my input pr? it's coming up now 😄